service/code_interpreter.py

```diff
@@ -73,36 +82,65 @@ async def __aexit__(self, _exc_type, _exc_value, _traceback):
             self.sandbox.keep_alive(self.timeout)
         self.sandbox.close()
 
-    def get_files_code(self):
+    def get_dataframe(self):
         """
         Get the code to read the files in the sandbox.
         This can be used for instructing the LLM how to access the loaded files.
         """
-
         # TODO: Add support for xslx, json
-        files_code = "\n".join(
-            f'df{i} = pd.read_csv("{self._get_file_path(url)}") # {url}'
-            for i, url in enumerate(self.file_urls)
+        df = pd.read_csv(self.file_urls[0])
+        return df, self.file_urls[0]
+
+    def genreate_prompt(self, query: str) -> str:
```

Comment:
Typo in `generate_prompt`
```suggestion
    def generate_prompt(self, query: str) -> str:
```